hydra/test_utils/config_source_common_tests.py

```diff
@@ -55,10 +72,35 @@ def test_is_group(
     )
     def test_is_config(
         self, type_: Type[ConfigSource], path: str, config_path: str, expected: bool
+    ) -> None:
+        self.is_config_helper(type_, path, config_path, expected)
+
+    @mark.parametrize(  # type: ignore
+        "config_path, expected",
+        [
+            ("dataset", True),
+        ],
+    )
+    def test_is_config_with_overlap_name(
+        self, type_: Type[ConfigSource], path: str, config_path: str, expected: bool
+    ) -> None:
+        if self.skip_overlap_config_path_name():
+            pytest.skip("Plugin does not support overlap config name with config path.")
+        self.is_config_helper(type_, path, config_path, expected)
+
+    def list_helper(
+        self,
+        type_: Type[ConfigSource],
+        path: str,
+        config_path: str,
+        results_filter: Optional[ObjectType],
+        expected: List[str],
     ) -> None:
         src = type_(provider="foo", path=path)
-        ret = src.is_config(config_path=config_path)
-        assert ret == expected
+        ret = src.list(config_path=config_path, results_filter=results_filter)
+        for x in expected:
+            assert x in ret
+        assert ret == sorted(ret)
```

Comment:
those helpers are adding more code than they are helping you reduce.
maybe just copy repeat the few important lines in it twice?

Reply:
😅 ok sounds good.